packages/build/src/steps/get.ts

```diff
@@ -39,7 +39,18 @@ export const getDevSteps = function (command, steps, eventHandlers?: any[]) {
 
   const eventSteps = getEventSteps(eventHandlers)
 
-  const sortedSteps = sortSteps([preDevCleanup, ...steps, eventSteps, devCommandStep], DEV_EVENTS)
+  const sortedSteps = sortSteps(
+    [
+      preDevCleanup,
+      ...steps,
+      // Trigger the uploadBlobs step during development to allow users to test file-based blob
+      // uploads locally and to allow frameworks to capture file-based blobs written by frameworks
+      { ...uploadBlobs, event: 'onDev' },
```

Comment:
I see that you're reusing the core step we have for builds and just changing the `event` property. As far as I know, we don't really have a precedent for running the same core step in multiple events, but I would probably have two separate core steps — one for dev, one for build — that may reuse common functionality. I think taking a core step and creating another one from it on-the-fly here might be confusing to people.

More importantly, if we run the existing core step as is as part of dev, won't we be uploading blobs to the actual production store? How do we ensure that we're uploading blobs to the local store?